time sent_time
VehicleState cur_vehicle_state
VehicleState front_vehicle_state
VehicleState back_vehicle_state
VehicleState[] adjacent_lane_vehicles
Pedestrian nearest_pedestrian
int32 max_num_vehicles
float64 speed_limit
Lane current_lane
Lane next_lane
RewardInfo reward
